### ISSUE TITLE
npm run auth isn't working

### ISSUE THREAD

**Issue by ArnavPC** [Fabrizz/MMM-OnSpotify#16]
I recently installed magic mirror2 on pop os.
It installed successfully. 
i was interested in this wonderful looking module.
it also successfully installed but when i run npm run auth it shows this error
npm ERR! Missing script: "auth"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/arnav/.npm/_logs/2023-05-23T10_43_34_330Z-debug-0.log

**Comment by Fabrizz** [comment]
Were are you performing the `npm run auth` command? You should be doing it inside the MMM-OnSpotify folder

**Comment by ArnavPC** [comment]
Yup I am doing that, in fact I just configured the calendar to display Indian holidays and configured some compliments and it works fine

**Comment by Fabrizz** [comment]
> Yup I am doing that, in fact I just configured the calendar to display Indian holidays and configured some compliments and it works fine

You are right for some reason I deleted the npm command (? Idk, I'm adding it now

**Comment by ArnavPC** [comment]
Or reinstall the module entirely

**Comment by Fabrizz** [comment]
It should work now

**Comment by ArnavPC** [comment]
well one problem goes another one arised i guess, so this is recurring problem and i have tested it several times, so it connects to my account at first startup and displays my albums and things, whenever i play a song it doesn't load it. when i restart the the mirror it says "Connecting" and never ever connects

if u need logs or whatever i will provide them
also i am using a pihole, but i disabled it and it still doesnt seem to work
